store/store.go

```diff
@@ -174,6 +175,15 @@ type Store struct {
 	suggestedCurrency string
 }
 
+func defaultRetryStrategy() retry.Strategy {
+	return retry.LimitCount(6, retry.LimitTime(30*time.Second,
```

Comment:
I'd suggest:

   var defaultRetryStrategy = ....

because we can always re-use the same strategy object  - no need to make it again each time.